admin-ui/plugins/admin/components/Webhook/ShortcodePopover.js

```diff
@@ -0,0 +1,160 @@
+import * as React from 'react'
+import Popover from '@mui/material/Popover'
+import Typography from '@mui/material/Typography'
+import Button from '@mui/material/Button'
+import Box from '@mui/material/Box'
+import Divider from '@mui/material/Divider'
+import { List, ListItemButton, ListItemText } from '@mui/material'
+import { useTranslation } from 'react-i18next'
+import { HelpOutline } from '@mui/icons-material'
+import { Tooltip as ReactTooltip } from 'react-tooltip'
+import PropTypes from 'prop-types'
+
+const Icon = () => (
+  <svg
+    xmlns='http://www.w3.org/2000/svg'
+    fill='none'
+    viewBox='0 0 24 24'
+    strokeWidth='1.5'
+    stroke='currentColor'
+    style={{ width: '18px' }}
+  >
+    <path
+      strokeLinecap='round'
+      strokeLinejoin='round'
+      d='M17.25 6.75 22.5 12l-5.25 5.25m-10.5 0L1.5 12l5.25-5.25m7.5-3-4.5 16.5'
+    />
+  </svg>
+)
+
+export default function ShortcodePopover({
+  codes,
+  buttonWrapperStyles,
+  handleSelectShortcode,
+}) {
+  const [anchorEl, setAnchorEl] = React.useState(null)
+  const handleClick = (event) => {
+    setAnchorEl(event.currentTarget)
+  }
+
+  const handleClose = () => {
+    setAnchorEl(null)
+  }
+
+  const open = Boolean(anchorEl)
+  const id = open ? 'simple-popover' : undefined
+
+  const wrapperStyles = {
```

Comment:
Can we keep images and CSS separately

Reply:
css moved to `app/routes/Apps/Gluu/styles/applicationstyle.js`